cmd/devstream/show.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/devstream-io/devstream/internal/pkg/show"
+	"github.com/devstream-io/devstream/pkg/util/log"
+)
+
+var plugin string
+
+var showCMD = &cobra.Command{
+	Use:   "show",
+	Short: "Show is used to print some useful information",
+	Long: `Show is used to print some useful information. 
+eg.
+- dtm show config --plugin=A-PLUGIN-NAME`,
```

Comment:
The correct abbreviation of "for example" is "e.g.", and since this is the beginning of a sentence, it should be:

"E.g.:"

But making the first letter uppercase looks weird, so maybe put "Examples:" instead.

Also, "-" can't be rendered as a bullet point like in markdown. so maybe we should use two spaces instead of a bullet.

```shell
Examples:
  dtm show config --plugin=argocd
```

Reply:
updated at list/show/develop